frontend/src/components/viewers/MetricsDropdown.test.tsx

```diff
@@ -409,5 +429,47 @@ describe('MetricsDropdown', () => {
     screen.getByTitle('run1 > execution1 > artifact1');
   });
 
-  // TODO: Namespace...
```

Comment:
I had already added a namespace test on the previous PR (https://github.com/kubeflow/pipelines/pull/8026#discussion_r926990665), but failed to fill in or remove this TODO. The new added tests below do not have to do with the namespace test.

Reply:
There are no longer added tests on this file due to this discussion: https://github.com/kubeflow/pipelines/pull/8029#discussion_r928339283.